index.js

```diff
@@ -14,22 +14,104 @@ const inspire = [
     '\"Went the distance, now I\'m back on my feet. Just a man and his will to survive.\" - Survivor',
     '\"There\'s vomit on his sweater already, mom\'s spaghetti\" - Eminem',
     '\"All our dreams can come true, if we have the courage to pursue them.\" – Walt Disney',
-    '\"\" – ',//12
-]
+    '\"Only the paranoid survive.\" – Andy Grove ',
+    '\"It’s hard to beat a person who never gives up.\" – Babe Ruth',
+    '\"It always seems impossible until it\'s done.\" - Nelson Mandela',
+    '\"Problems are not stop signs, they are guidelines.\" - Robert H. Schuller',
+    '\"It\'s Not Whether You Get Knocked Down, It\'s Whether You Get Up.\" - Vince Lombardi',
+    '\"We may encounter many defeats, but we must not be defeated.\" - Maya Angelou',
+];
 
+const mock = [
```

Comment:
Eventually want to move the mock quotes to a service call or feed from somewhere.